test/utils.ts

```diff
@@ -41,6 +44,22 @@ export const waitForSpy = (spy: SinonSpy, property = 'called') => {
   });
 };
 
+/**
+ * Creates and returns an os-specific directory for temp files.
+ */
+export const getTempDir = (unique: boolean) => {
+  let dir = path.join(os.tmpdir(), 'xud-test');
+  if (unique) {
+    dir = path.join(dir, uuidv1());
+  }
+
+  if (!fs.existsSync(dir)) {
```

Comment:
Minor nit, but you can just `try` the mkdir call and in the `catch` ignore the exception if `err.code === 'EEXIST'`. That will just combine two calls into one and make the tests run ever so slightly faster.

Also you can consider using the async version of the mkdir call that returns a promise, but I figure the tests are being run one at a time anyway so I'm thinking it probably wouldn't make a difference.

Reply:
I noticed that due to the `{recursive: true}` option, that error isn’t thrown. 
Assuming that the implementation is different on that option, I wasn’t sure if removing `existsSync` would cause performance gain, so I left it as it. 

In regards to the async version - I thought it would just create a hassle for the test code.